src/Console/stubs/directives/arg_builder_directive.stub

```diff
@@ -0,0 +1,11 @@
+    /**
+     * Add additional constraints to the builder based on the given argument value.
+     *
+     * @param  \Illuminate\Database\Query\Builder|\Illuminate\Database\Eloquent\Builder  $builder
+     * @param  mixed  $value
+     * @return \Illuminate\Database\Query\Builder|\Illuminate\Database\Eloquent\Builder
+     */
+    public function handleBuilder($builder, $value)
+    {
+        // TODO implement the arg builder
+    }
```

Comment:
Nit: having a newline here would be nice, it plays better with git and other tools.